src/GafferArnold/ArnoldOptions.cpp

```diff
@@ -100,7 +100,7 @@ ArnoldOptions::ArnoldOptions( const std::string &name )
 
 	options->addOptionalMember( "ai:texture_searchpath", new IECore::StringData( "" ), "textureSearchPath", Gaffer::Plug::Default, false );
 	options->addOptionalMember( "ai:procedural_searchpath", new IECore::StringData( "" ), "proceduralSearchPath", Gaffer::Plug::Default, false );
-	options->addOptionalMember( "ai:shader_searchpath", new IECore::StringData( "" ), "shaderSearchPath", Gaffer::Plug::Default, false );
+	options->addOptionalMember( "ai:plugin_searchpath", new IECore::StringData( "" ), "pluginSearchPath", Gaffer::Plug::Default, false );
```

Comment:
Should we add a compatibility config for this?

Reply:
From an IE point of view, we're not really expecting old scenes to continue working ( and artists aren't overriding this path anyway ).  Do you think there is someone else out there who benefit enough to be worth worrying about?

Reply:
I dunno to be honest - if anyone out there would like this, let us know! In the absence of input, perhaps we do nothing for now, knowing we can always implement one after the fact if we find a need?